docs/CHANGELOG.rst

```diff
@@ -1,321 +1,484 @@
-.. _changelog:
+`v9.2 <https://github.com/lvgl/lvgl/compare/v9.1.0...v9.2>`__ 26 August 2024
```

Comment:
Make sure the tag actually gets named "v9.2" and not "v9.2.0", otherwise, change this line to say "v9.2.0"